common/utils.go

```diff
@@ -63,3 +65,17 @@ func GunzipData(data []byte) ([]byte, error) {
 
 	return resB.Bytes(), nil
 }
+
+func RemoveGlob(path string) error {
```

Comment:
Question:- Does it make sense to have unit tests for this?

Reply:
It just call system function, let's have some confidence about it.